.github/workflows/ci.yml

```diff
@@ -12,25 +12,17 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ '1.19', '1.18', '1.x' ]
-        gostable: [true]
-        include:
-        - go: '1.x'
-          gostable: false
+        go: [ '1.19', '1.18']
```

Comment:
nit, but spaces are inconsistent. I agree with Jeff that I like having 1.x automatically pick the newest version. I'm not a big fan of using `stable` or `oldstable` since they don't necessarily match the policy that we have. Can we revert that line specifically?

Reply:
Ok, I reverted that line.

I also don't like `stable` and `oldstable` very much; I prefer having control on the Go versions used by CI.
However, by keeping both `1.19` and `1.x`, we're wasting CI compute resource :seedling: